### PR TITLE
meta: add new packages.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

File: abb_experimental/package.xml [hunk 1]
```diff
@@ -18,9 +18,12 @@
...
   <exec_depend>abb_irb120_support</exec_depend>
   <exec_depend>abb_irb120t_moveit_config</exec_depend>
+  <exec_depend>abb_irb1600_support</exec_depend>
   <exec_depend>abb_irb2600_support</exec_depend>
   <exec_depend>abb_irb4600_support</exec_depend>
   <exec_depend>abb_irb52_support</exec_depend>
+  <exec_depend>abb_irb6650s_support</exec_depend>
+  <exec_depend>abb_irb7600_support</exec_depend>
 
   <export>
```
Comment (AustinDeric) [inline abb_experimental/package.xml:28]: maybe add <architecture_independent/> ?

https://github.com/ros-infrastructure/rep/blob/240583e7c1b5148e0dc939bd40a628fda047aa26/rep-0140.rst#architecture_independent

"Specifying <architecture_independent/> is recommended for metapackages..."

**Review by AustinDeric — CHANGES_REQUESTED** [review]

**Comment by AustinDeric** [comment]
These new packages add pressure to a solution for #46, transitioning robots to abb.

**Comment by gavanderhoorn** [comment]
@AustinDeric wrote:
> These new packages add pressure to a solution for #46, transitioning robots to abb.

why exactly?

File: abb_experimental/package.xml (hunk 1 above)
Comment (gavanderhoorn) [reply abb_experimental/package.xml:28]: Missed that one.

Done now.

**Review by AustinDeric — APPROVED** [review]

**Comment by AustinDeric** [comment]
+1 Thank you!

**Pull request merged**